main.js

```diff
@@ -218,8 +218,19 @@ const template = [
           await shell.openExternal('https://rapidpowerestimator.readthedocs.io/en/latest/');
         },
       },
+      {
+        label: 'About',
+        click: () => {
+          dialog.showMessageBox(mainWindow, {
+            type: 'info',
+            title: 'Rapid Power Estimator',
+            message: 'Rapid Power Estimator\nVersion: 0.0.1\nAuthor: © 2024 Rapid Silicon Inc. \nType: Engineering',
```

Comment:
We should not hardcode this, it must be read the from package json file.

Reply:
Done, We have RPE name as rapid_power_estimator in package.json, is there a way we can change it to 'Rapid Power Estimator'?

Reply:
Read from these keys instead ?

  "productName": "Rapid Power Estimator",
    "copyright": "©2024 Rapid Silicon, Inc.",